lib/staticizer/crawler.rb

```diff
@@ -164,6 +164,16 @@ def save_page_to_aws(response, uri)
       # Upload this file directly to AWS::S3
       opts = {:acl => "public-read"}
       opts[:content_type] = response['content-type'] rescue "text/html"
+
+      # Detect a meta-redirect and set an S3 hosting redirect metadata item
+      if response =~ /META http-equiv='refresh' content='0;URL="(.*)"/
+        location = $1
+        if location =~ /^(?:[^\/]|http:\/\/|https\:\/\/).*/
```

Comment:
This is prepending a slash if the location starts with http or https, is that needed for s3 redirects? Also wouldn't this redirect to the wrong place if the location is not absolute. So if we are at http://www.google.com/section/page1 and that page has a meta refresh to url='page2' then this would redirect to /page2 instead of /section/page2.